test/shared/src/main/scala/zio/test/ZIOSpecAbstract.scala

```diff
@@ -74,19 +73,13 @@ abstract class ZIOSpecAbstract extends ZIOApp with ZIOSpecAbstractVersionSpecifi
       testArgs = TestArgs.parse(args.getArgs.toArray)
       summary <- runSpecAsApp(spec, testArgs, console)
       _ <- ZIO.when(testArgs.printSummary) {
```

Comment:
@hmemcpy I think we can remove this guard now, since we're always going to use the correct `TestRenderer`, and the `IntellijTestRenderer` will just be an empty string.